src/components/radioButtonCO.js

```diff
@@ -10,7 +10,7 @@ import palette from "../theme/palette"
 const useStyles = makeStyles({
   div: {
     width: 180,
-    height: 40,
+    height: 32,
     "& .Mui-focusVisible": {
```

Comment:
I think you are doing wrong class attribute, because you are using the **Global class**  and not the **Rule name**

For example: 
```css
focusVisible : {
....
}
```
Pseudo-class applied to the `RadioButton` root of element